services/fishAges.service.ts

```diff
@@ -43,6 +49,17 @@ export type FishAge<
     scopes: {
       ...COMMON_SCOPES,
     },
+    actions: {
+      remove: {
+        types: [RestrictionType.ADMIN],
+      },
+      create: {
+        types: [RestrictionType.ADMIN],
+      },
+      update: {
+        types: [RestrictionType.ADMIN],
+      },
```

Comment:
disable rest for other actions that are not in use (see console for list of actions/rest endpoints) OR enable restrictions as well